compose.go

```diff
@@ -106,6 +162,14 @@ func (dc *LocalDockerCompose) WithEnv(env map[string]string) DockerCompose {
 	return dc
 }
 
+// WithExposedService sets the strategy for the service that is to be waited on. If multiple strategies
+// are given for a single service running on different ports, both strategies will be applied on the same container
+func (dc *LocalDockerCompose) WithExposedService(service string, port int, strategy wait.Strategy) DockerCompose {
```

Comment:
Wdyt about renaming this function to `WaitingFor(service, port, strategy)`?

I'd also document why we need the port at this API method, as implemented in the `applyStrategyToRunningContainer`

Reply:
Thinking about it out loud... Should a consumer waiting for a log entry pass the port? I'd like to simply pass the strategy and then the code should infer what to do there. Do you see any other way to filter/get the service container without passing the port?